source/storage/apis-1.5.rst

```diff
@@ -432,6 +508,47 @@ Request Headers
     response will be returned.
 
 
+**X-Weave-Records**
+
+    This header may be sent with multi-record uploads, to indicate the
```

Comment:
I assume all these headers only make sense with batch=true (ie, for the first post rather than subsequent ones)?